src/vcpkg/binarycaching.cpp

```diff
@@ -1255,6 +1255,159 @@ namespace
         Path m_tool;
     };
 
+    struct AzureUpkgTool
+    {
+        AzureUpkgTool(const ToolCache& cache, MessageSink& sink) { az_cli = cache.get_tool_path(Tools::AZCLI, sink); }
+
+        Command base_cmd(const AzureUpkgSource& src,
+                         StringView package_name,
+                         StringView package_version,
+                         StringView az_cmd) const
```

Comment:
`az_cmd` sounds like it's supposed to be a command line but it's escaped into a single parameter on 1268; is that the intent?

How about "verb"?